src/Faker/Provider/Text.php

```diff
@@ -127,6 +128,7 @@ protected static function strlen($text)
 
     protected static function validStart($word)
     {
+        if(  !  $this->textStartsWithUppercase ) return true;
```

Comment:
Have you considered using https://styleci.io/ @fzaninotto? It would mark builds as failing in the same way travis does on pull requests. Would save you commenting on pulls for such cases.